### ISSUE TITLE
Video Display Abnormal

### ISSUE THREAD

**Issue by 1eooyang** [androidx/media#382]
### Media3 Version

ExoPlayer 2.16.0

### Devices that reproduce the issue

OPPO PDYM20 running Android 12

### Reproducible in the demo app?

Not tested

### Reproduction steps

Video link is https://online-education.codemao.cn/228/codecamp/e7e744a1933747c5ada76ca79e7fced6.mp4，all mentioned link to the video below indicate this link 
1. Use Exoplayer to run the video link;
2. Use Chrome to run the video link;

### Expected result

1. Using Exoplayer to run this link is normal
2. Using Chrome to run this link is normal

### Actual result

1. Using Exoplayer, it's stalling a few second at the begining of this video;
2. Using Chrome, it's playing normal

### Media

https://user-images.githubusercontent.com/20480296/236613031-5a8a7d94-6e85-4460-bfea-30a788d638b8.mp4

https://user-images.githubusercontent.com/20480296/236613094-9dd68d03-ee84-4f2f-860b-fd6d633fd39f.mp4

### Bug Report

- [ ] You will email the zip file produced by `adb bugreport` to dev.exoplayer@gmail.com after filing this issue.

**Comment by 1eooyang** [comment]
Has anyone dealt with this issue?

**Comment by microkatz** [comment]
Hi @1eooyang. Thank you for reporting your issue!

Would you please share a [bug report](https://developer.android.com/studio/debug/bug-report) so that we can review ExoPlayer logs during your scenario? If you cannot share them publicly then please send them to dev.exoplayer@gmail.com using a subject in the format "Issue #382". Please also update this issue to indicate you've done this.

**Comment by 1eooyang** [comment]
[exolog.txt](https://github.com/androidx/media/files/11571208/exolog.txt)
@microkatz  here is the exopalyer logs

**Comment by microkatz** [comment]
Hi @1eooyang.

Thank you for your patience. 

I was able to test and reproduce a delay. However, it appears to be a network issue of retrieving the mp4 file. If I downloaded the video unto the phone then I could not reproduce the delay at all. The difference of your browser vs your phone is perhaps different network capabilities.

**Comment by 1eooyang** [comment]
I use  phone and  chrome to compare it just to demonstrate the problem. In actual application scenarios, network video is often used, but exoplayer will have problems, the system mediaplayer will not, neither will ijkplayer, and neither will chrome. I want to know why , and how to fix this problem.

Forgive my poor English.

**Comment by 1eooyang** [comment]
@microkatz

**Comment by 1eooyang** [comment]
And only a few special links have problems, most of the links are normal

**Comment by 1eooyang** [comment]
@microkatz  yes , it still have the same delay issue when i  play  the video i downloaded .
